fatoptimizer/config.py

```diff
@@ -1,7 +1,8 @@
 import builtins
 
 from .tools import get_constant_size, ITERABLE_TYPES
-
+# ISSUE : SystemError: Parent module '' not loaded, cannot perform relative import
+# StackOverFlow search : http://stackoverflow.com/questions/16981921/relative-imports-in-python-3
```

Comment:
I don't understand why you added this comment. Your probably misused Python. Maybe you tried to import a module whereas you were on the wrong directory. To be clear: the code works very well. All unit tests pass, so you must not add such comment.